lib/uwb/UwbMacAddress.cxx

```diff
@@ -25,6 +27,43 @@ UwbMacAddress::UwbMacAddress(const UwbMacAddress& other) :
     InitializeView();
 }
 
+UwbMacAddress::UwbMacAddress(std::string addressString, UwbMacAddressType addressType)
```

Comment:
Can you please add tests for this [here](https://github.com/microsoft/nearobject-framework/blob/develop/tests/unit/uwb/TestUwbMacAddress.cxx)?

Reply:
Yes, I added tests locally and it helped me find a bug.. so I will push new changes once fixed!